### PR TITLE
Modified Realistic People/House ratio

### PR DESCRIPTION
**Pull request by manandey**

### CODE REVIEW

**Comment by manandey** [comment]
[Private households by household size.docx](https://github.com/pg2455/covid_p2p_simulation/files/4456249/Private.households.by.household.size.docx)

File: run.py [hunk 1]
```diff
@@ -170,14 +171,27 @@ def run_simu(n_stores=None, n_people=None, n_parks=None, n_misc=None,
...
+        new_dict[i.name] = 0, i.capacity
+
+    def _assign_household(rng):
+        household = rng.choice(households)
+        occ, cap = new_dict[household.name]
+        if (occ < cap):
+            new_dict[household.name] = occ + 1, cap
+        else:
+            _assign_household(rng)
+        return household
```
Comment (pg2455) [inline run.py]: Given that it is not taking  care of age, how does it ensure that 2 2year olds are not housed in a house of cap 2? 

I might be missing something here. Sorry about that!

**Review by pg2455 — COMMENTED** [review]
Just a minor concern regarding the age of people in any house

File: run.py (hunk 1 above)
Comment (manandey) [reply run.py]: This PR is for Realistic People/House Ratio (Task - 89), Will submit a separate PR for the age distribution problem shortly.

Comment (manandey) [reply run.py]: Hi Prateek, Modified taking age into consideration as well as people/house ratio.  Can you once have a look into it?

**Comment by mweiss17** [comment]
I think we have incorporated this functionality now -- if not, please resubmit this PR against the develop branch.

**Pull request closed without merging**